BasicSample/Assets/GrabAndThrow/Scripts/GrabAndThrowManager.cs

```diff
@@ -53,12 +53,29 @@ void Update()
 			UpdateForHand(XRNode.RightHand, m_rightHandData);
 		}
 
+		private bool? TryGetIsTapping(InputDevice device)
```

Comment:
Both of these bindings, "PrimaryButton" and "Squeeze" (shows up for me in the Interactions scene as "Grip") work on the HL2, they just have slightly different conditions for triggering. PrimaryButton detects a pinching gesture, like an air tap; Squeeze/Grip detects any closed hand gesture - which does include the pinching gesture, but also includes making a fist.

Switching from PrimaryButton to Squeeze/Grip would allow you to grab the virtual objects with a closed fist in addition to with a pinch.

Reply:
the primary different of the "primary button" on HL2, is that the airtap must happen when hand is facing down.  User cannot do a "airtap" when hand is facing up.   The "squeeze" action is a much better match to this grab scenario.

Reply:
Enable the "grab" using feature usages of "TriggerButton", "GripButton", and "PrimaryButton".

Reply:
addressed with latest commits